examples/12. Advanced Action.md

```diff
@@ -0,0 +1,66 @@
+FORMAT: 1A
+
+# Advanced Action API
+An action in a resource in the API Blueprint can be considered as a state transition. This API example demonstrates that a resource can contain a state transition to another resource. This allows us to show that the resource `Task` can afford an action that is not limited to the resource `Task`.
+
+## API Blueprint
++ [Previous: Resource Model](11.%20Resource%20Model.md)
++ [This: Raw API Blueprint](https://raw.github.com/apiaryio/api-blueprint/master/examples/11.%20Advanced%20Action.md)
+
+# Task [/tasks/{id}]
+
++ Parameters
+    + id (string)
+
+## Retrieve Task [GET]
+
++ Response 200 (application/json)
+
+        {
+            "id": 123,
+            "name": "Go to gym",
+            "done": false,
+            "type": "task"
+        }
+
+## Update Task [PATCH]
+This is a state transition within the same resource
```

Comment:
Either way I think we are in an agreement here. There are two "states" definitions: 

1. a state is one particular combination of data values and available affordances
2. a state is one particular combination of available affordances regardless of data values

So I believe originally I was talking about No2 while you made your point on No1. It would is good to figure this one out!